feat: simplified usage without writing streamlit apps (#127)
pasteurlabs/tesseract-streamlit: README.md

## 🧰 Usage

```bash
$ tesseract-streamlit [OPTIONS] [URL] [OUTPUT]
```

* `URL`: The address to the Tesseract instance you want to interface with. Not needed with `--from-image`.
* `OUTPUT`: (Optional) The file path to write the generated Streamlit app. When omitted, the app is written to a cache file and Streamlit is launched automatically.

### Quick start

> The fastest way to get started — just point at a running Tesseract:

Reviewer: ```suggestion
If you've got a running Tesseract, you can get started fast by simply passing in its URL.
```

```bash
$ tesseract-streamlit http://127.0.0.1:61725
```

This generates the app and launches Streamlit in one step.

Or, skip serving the Tesseract yourself entirely:

```bash
$ tesseract-streamlit --from-image vectoradd
```

> This serves the Tesseract from the given image, generates the app, and launches Streamlit — all in one command.

Reviewer: ```suggestion
This serves the Tesseract from the given image, generates the app, and launches Streamlit, in a single command.
```
